style.css

```diff
@@ -0,0 +1,43 @@
+body{
+    background-image: url(./image/photo-1506755594592-349d12a7c52a.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    background-attachment: fixed;
+  
+}
+.keys{
+    display: flex;
+   
+  }
+ .key{
+     justify-content: center;
+     border: 4px solid white;
+     border-radius: 5px;
+     margin: 10px;
+     font-size: 15px;
+     padding: 10px 10px;
+     transition: all 0.07s;
+     width: 100px;
+     text-align: center;
+     color: black;
+     background-color: lightblue;
+     text-shadow: 0 0 5px  black;
+     margin-top: 30%;
+ 
+ }
+ .playing{
```

Comment:
well, yes) here we have a problem, pls review animate keys miestone in the repo of Evan, .playing style is a part of my user story, as far as this is the animated style

Reply:
but i think see beauty is all about style. so what do you think ? should i have delete?

Reply:
https://github.com/HackYourFutureBelgium/drum-kit/issues/17
I'm just following the steps in evan's repo, as agreed. So if you write it , I have nothing to do on this step, pls check the link

Reply:
ok no problem i will remove .